labgrid/driver/power/phoenix_fl_switch.py

```diff
@@ -0,0 +1,80 @@
+'''
+This Driver was tested on a FL SWITCH 2303-8SP1 with FW-version 3.27.01 BETA
+file   phoenix_fl_switch.py
+author Raffael Krakau
+date   2023-08-24
+
+Copyright 2023 JUMO GmbH & Co. KG
+'''
+
+from telnetlib import Telnet
+
+PORT=23
+
+def power_set(host, port, index: int, value: bool):
+    """
+    Set power state by socket port number (e.g. 1 - 8) and an value {'enable', 'disable'}.
+
+    - values:
+        - disable(False): Turn OFF,
+        - enable(True): Turn ON
+    """
+    username = "admin"
+    password = "private"
+    action = "enable" if value else "disable"
+
+    telnet = Telnet(host=host, port=port)
+
```

Comment:
I would suggest to use the follow lines with the `with` statement: 
 (https://docs.python.org/3/library/telnetlib.html)
```
with Telnet('localhost', 23) as tn:
    tn....
    tn...
    . 
    . 


```
further it might be good to have it encapsulated in a helper function

Reply:
besides using pexpect done

Reply:
:+1: